sources/packages/api/src/route-controllers/designation-agreement/models/designation-agreement.model.ts

```diff
@@ -0,0 +1,34 @@
+import { DesignationAgreementStatus } from "../../../database/entities";
+
+/**
+ * This DTO contains dynamic data that must
+ * be validated by the form.io dryrun validation.
+ */
+export interface SubmitDesignationAgreementDto {
+  dynamicData: any;
```

Comment:
Is it the form.io data? we can have an interface here?

Reply:
The data is going to be the result of an area in the form.io that is not required to have any constraints and could be freely changed by the Ministry. The data under this particular section is also not supposed to be used anywhere in the API, only to populate the same form again for visualization, so, I do not see a reason for this particular case.